test/unit/vdom/incremental-dom.js

```diff
@@ -1,8 +1,34 @@
 import { IncrementalDOM as VdomIncrementalDOM } from '../../../src/api/vdom';
-import IncrementalDOM from 'incremental-dom';
+import * as IncrementalDOM from 'incremental-dom';
+import element from '../../lib/element';
+import { symbols, vdom } from '../../../src/index';
 
 describe('IncrementalDOM', function () {
   it('should export all the same members as the incremental-dom we consume', function () {
     expect(VdomIncrementalDOM).to.contain(IncrementalDOM);
   });
 });
+
+describe('VdomIncrementalDOM', function () {
+  describe('attributes (default)', function () {
+    it('should not set properties on SVG elements', function () {
+      // This should throw but the error is different depending on the browser,
+      // so we don't expect any particular message.
```

Comment:
If we're expecting an error everywhere, why not check it with `expect.to.throw` ?
```
expect(function() {
          //... func body ...
        }).to.throw(Error);
```